build.gradle.kts

```
@@ -195,8 +195,20 @@ tasks {
 			if (modularBuild.toBoolean())
 				java.srcDir("src/main/module")
 		}
+		create("demo") {
+			java {
+				srcDir("src/demo/java")
+			}
+			compileClasspath += sourceSets.main.get().output
+			runtimeClasspath += sourceSets.main.get().output
+		}
 	}
 
```

```suggestion
	project(":demo") {
		sonarqube {
			isSkipProject = true
		}
	}
```

this should do the trick :)